cmd/zfs/zfs_main.c

```diff
@@ -4417,13 +4474,17 @@ zfs_do_send(int argc, char **argv)
 		{"backup",	no_argument,		NULL, 'b'},
 		{"holds",	no_argument,		NULL, 'h'},
 		{"saved",	no_argument,		NULL, 'S'},
+		{"exclude",	required_argument,	NULL, 'X'},
```

Comment:
How about `-x` which would be more consistent?  Historically we haven't resorted to capital letters unless there's a conflict (with perhaps a couple exceptions).  Or is there one I missed?

```suggestion
		{"exclude",	required_argument,	NULL, 'x'},
```

Reply:
I have absolutely no preference, so making the change.

Reply:
I actually misread this as using -x initially and was going to request it be -X or something else before I noticed I was mistaken, just because having send and recv use -x for different things seemed prone to mishap.

But perhaps I'm just overthinking it.

Reply:
Good catch.  Yes, that's exactly the kind of confusion I was hoping to avoid, I'd forgotten `-x` was used by `zfs receive`.  In that case, I'd be good with `-X` or something else.